imap_processing/tests/swe/test_swe_l1b.py

```diff
@@ -78,63 +77,24 @@ def test_swe_l1b(decom_test_data):
     for field in science_eu_field_list:
         assert round(science_l1b[field].data[1], 5) == round(validation_data[field], 5)
 
-    # process housekeeping data to l1a and create l1b
-    hk_l1a_ds = create_dataset(grouped_data[1330])
-    hk_l1b = convert_raw_to_eu(
-        hk_l1a_ds, conversion_table_path, SWEAPID.SWE_APP_HK.name
-    )
-
-    # read housekeeping validation data
-    eu_validation_data = pd.read_csv(
-        test_data_path / "data_derived.SWE_APP_HK_20230927_094839.csv",
-        index_col="SHCOARSE",
-    )
-    second_data = grouped_data[1330][1]
-    validation_data = eu_validation_data.loc[second_data.data["SHCOARSE"].raw_value]
-
-    # check that these field's calculated EU value matches with
-    # validation data's EU value.
-    eu_field_list = [
-        "HVPS_VBULK",
-        "HVPS_VCEM",
-        "HVPS_VESA",
-        "HVPS_VESA_LOW_RANGE",
-        "HVPS_ICEM",
-        "FEE_TEMP",
-        "SENSOR_TEMP",
-        "HVPS_TEMP",
-        "LVPS_BACK_BOARD_TEMP",
-    ]
-
-    # Check EU values from housekeeping data
-    for field in eu_field_list:
-        assert round(hk_l1b[field].data[1], 5) == round(validation_data[field], 5)
-
 
+@pytest.mark.xfail(reason="Will update test data with multiple packets")
```

Comment:
Does L1B need multiple packets? It looks like you are just validating filenames below, so I would have expected this to still work.

Reply:
I can't create l1b CDF file if I have any good processed data. It takes 4 packets to create one good cycle data. In upcoming PR, Yaxi gave me bin file with multiple good packets data. I plan to update this in upcoming PR. small PR with intention